packages/wrangler/src/hyperdrive/index.ts

```diff
@@ -177,9 +185,9 @@ export function getOriginFromArgs<
 			throw new UserError(
 				"You must provide a port number - e.g. 'user:password@database.example.com:port/databasename"
 			);
-		} else if (url.pathname === "") {
+		} else if (!url.pathname || url.pathname === "") {
```

Comment:
If the pathname is the empty string then `!url.pathname` is true.

```suggestion
		} else if (!url.pathname) {
```